Sealnote/src/main/java/com/twistedplane/sealnote/data/Note.java

```diff
@@ -93,7 +93,7 @@ private void readFromParcel(Parcel inParcel) {
         mArchived = inParcel.readInt() > 0;
         mDeleted = inParcel.readInt() > 0;
         mType = Type.valueOf(inParcel.readString());
-        mNote = NoteContent.fromString(mType, inParcel.readString());
+        mNote = NoteContent.fromString(mType, note);
```

Comment:
The number of `writeString` and `readString` don't match up in the original code. `note` variable is never used. `NoteContent` must have the actual note with the type right ?